AppSettingsManagerAPI/AppSettingsManagerApi.Infrastructure/MySql/Converters/ServiceConfiguration.cs

```diff
@@ -7,18 +7,18 @@ public static class ServiceConfiguration
 {
     public static IServiceCollection AddConverters(this IServiceCollection services)
     {
-        services.AddSingleton<IBidirectionalConverter<Model.User, User>, UserBiverter>();
+        services.AddSingleton<IBidirectionalConverter<Model.User, User>, UserConverter>();
```

Comment:
Could probably run 'dotnet csharpier .' to keep formatting consistent across everything but if you don't have that tool installed don't worry about it. I think I'll have another update to the API and I can lint it then

Reply:
Or maybe this is linted and these couple of lines weren't long enough to be broken up